src/trusted_nodes.json

```diff
@@ -0,0 +1,161 @@
+[
```

Comment:
make this initial trusted nodes and what you call `extended trusted nodes` into `trusted nodes` (the filename)